screwdriver/screwdriver.go

```diff
@@ -169,18 +169,22 @@ type CommandDef struct {
 // Need a generic interface to take in an int or array of ints
 type IntOrArray interface{}
 
+type Stats struct {
```

Comment:
can we have this in-place defined inside the Build struct? do you see any issues?

Reply:
I fixed it